airflow-core/src/airflow/ui/src/pages/HITLTaskInstances/HITLTaskInstances.tsx

```diff
@@ -133,9 +133,11 @@ export const HITLTaskInstances = () => {
 
   const { data, error, isLoading } = useHumanInTheLoopServiceGetHitlDetails(
     {
-      dagIdPattern: dagId,
+      dagId,
       dagRunId: runId,
       responseReceived: Boolean(responseReceived) ? responseReceived === "true" : undefined,
+      taskId: Boolean(groupId) ? undefined : taskId,
+      taskIdPattern: groupId,
```

Comment:
Why do we do special treatment for group there?

As mentioned in the other PR this is a wrong way of filtering 'per group id' and I think we should remove that altogether for now until we update the backend.

Also 'Action Required' are not yet supported for group, I'm not sure why we have group code path here?

Reply:
Since this is not related to this PR, we can merge, and remove such code in a follow up PR.

Reply:
cc: @guan404ming